olGO/Views/Auth/RegisterView.swift

```diff
@@ -1,89 +1,156 @@
-                            Field(value: "", placeholder: "Password", keyboardType: .default)
-                                .configure {
-                                    $0.isSecureTextEntry = true
-                            }
-                            .inputHandler { (value) in
-                                self.password = value
-                            }
-                            .frame(height: 60)
-                        ]
-                    },
-                    Button("Register", titleColor: .blue) {
-                        self.register()
+    var body: some View {
+        let cornerRadius: CGFloat = 10
+        
+        return VStack {
+            HStack {
+                // Image calls on asset logo, may need to rename if asset name changes
+                Image("oneleifgoodlogo")
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width: 100, height: 100)
+                    .cornerRadius(cornerRadius)
+                Text("Sign Up for OneLeif!")
+                    .font(.title)
+                    .fontWeight(.bold)
+            }
+            .padding(.top, 10)
+            
+            HStack {
+                        Text("Email:       ")
+                        TextField(" \"John\" ", text: $email)
+                        .textFieldStyle(RoundedBorderTextFieldStyle())
+                    }
+                    
+                    HStack {
+                        Text("Password:")
+                        SecureField(" Check note below", text: $password)
+                        .textFieldStyle(RoundedBorderTextFieldStyle())
+                    }
+                    
+                    HStack {
+                        Text("Confirm:   ")
+                        SecureField(" Re-type Here", text: $confirmPassword)
+                        .textFieldStyle(RoundedBorderTextFieldStyle())
+                    }
+                    
+                    Text("Password must have at least 7 characters   ")
+                        .padding(.top, 50)
+                    
+                    HStack {
+                        VStack(alignment: .leading) {
+                            Text("At least 1 uppercase")
+                            Text("At least 1 number")
+                        }
+                        .padding(.trailing, 20)
+                        VStack(alignment: .leading){
+                            Text("At least 1 special")
+                            Text("At least 1 lowercase")
+                        }
                     }
-                    .frame(height: 60),
-                    Spacer()
-                ]
+                    .padding(.bottom, 50)
+                    
+                    Button("Submit") {
+                        
+                        self.submit()
+                        
+                    }.disabled(email.isEmpty || passwordProtocol())
```

Comment:
`.disabled(email.isEmpty || !passwordProtocol())`

Reply:
Crap I changed it so that if passwordProtocol() runs it throws an alert but this most recent commit I copy and pasted some code over that I should have I'll change it back so it gives the user an alert why they can't submit